### ISSUE TITLE
Latest Travis failure (Permission denied (publickey))

### ISSUE THREAD

**Issue by asmeurer** [gforsyth/doctr#41]
I'm not sure why Travis is failing now https://travis-ci.org/gforsyth/doctr/jobs/149058779

The error is

```
Build finished. The HTML pages are in _build/html.
Setting git attributes
git config --global user.name 'Travis docs builder (Travis CI)'
Adding doctr remote
git remote add doctr_remote git@github.com:gforsyth/doctr.git
Fetching doctr remote
git fetch doctr_remote
Warning: Permanently added the RSA host key for IP address '192.30.253.113' to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.
```

So something is wrong with the deploy ssh key. Maybe the stuff I'm putting in ~/.ssh/config, which was taken from https://github.com/alrra/travis-scripts/blob/master/doc/github-deploy-keys.md, is wrong. Other guides that I've seen use `ssh-add`. Do we need to use that. Is it the same? 

@gforsyth can you verify that the deploy key is there on GitHub?

**Comment by gforsyth** [comment]
@asmeurer It's there.  It does say that it's never been used

**Comment by gforsyth** [comment]
Fixed it!  We need to start the `ssh-agent` and then also add the key to the agent.  I made a bunch of dirty changes to figure out what was wrong, let me undo them and I'll put in a PR

**Comment by gforsyth** [comment]
Fixed in #44
